.github/workflows/on-pull-request.yml

```diff
@@ -0,0 +1,88 @@
+---
+# Full PR suite.  PRs need to pass this pipeline in order to be moved to the
+# merge queue, where they'll use the next rule as the branch-protection rule
+# for the final merge to the base branch.
+name: Pull Requests
+on:
+  pull_request:
+    branches: [main, "stable/*"]
+
+concurrency:
+  group: ${{ github.repository }}-${{ github.ref }}-${{ github.head_ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+jobs:
+  lint-docs:
+    if: github.repository_owner == 'Qiskit'
+    name: Docs and Lint
+    uses: ./.github/workflows/lint_docs.yml
+  main-tests-ubuntu-3-9:
+    name: Python Unit Tests
+    uses: ./.github/workflows/test-linux.yml
+    if: github.repository_owner == 'Qiskit'
+    with:
+      python-version: "3.9"
+      install-optionals: true
+      runner: ubuntu-latest
+  main-tests-rust:
+    name: Rust Unit Tests
+    uses: ./.github/workflows/rust-tests.yml
+    if: github.repository_owner == 'Qiskit'
+    with:
+      python-version: "3.9"
+      runner: ubuntu-latest
+  main-tests-images:
+    name: Image Comparison Tests
+    uses: ./.github/workflows/image-tests.yml
+    if: github.repository_owner == 'Qiskit'
+    with:
+      python-version: "3.9"
+      runner: ubuntu-latest
+  # The rest of the PR pipeline is to test the oldest and newest supported
+  # versions of Python.
+  main-tests-ubuntu-3-9-arm:
+    if: github.repository_owner == 'Qiskit'
+    name: Python Unit Tests
+    uses: ./.github/workflows/test-linux.yml
+    with:
+      python-version: "3.9"
+      install-from-sdist: true
+      install-optionals: false
```

Comment:
Shouldn't this be true to match the other 3.9 job? Or do the optional packages not all work on aarch64? (I vaguely remember that might have been the case).

Reply:
Fixed in 3a00da3, merged into a matrix job